release-team/role-handbooks/test-infra/README.md

```diff
@@ -0,0 +1,60 @@
+# Test Infra Lead Playbook
+
+## Overview of Test Infra Lead responsibilities
+
+Note: Currently, the test-infra lead has to be someone from Google GKE Engprod Team, in order to gain access to the prow cluster. This
+will change once we migrate our testing infrastructure under CNCF account.
+
+There are two major area that test-infra lead need to take care during the release cycle, which are:
+
+1. Create CI/Presubmit jobs for the new release, and polulate the testgrid dashboard
+2. Watch for test infra status, make sure test infra is stable, and react to test infra related issues
```

Comment:
Suggest:

2. Watch for test infra status, make sure test infra is stable, react to test infra related issues and notify Release Lead and CI Signal Lead of issue status changes

Reply:
accepted